doc/move_group_interface/src/move_group_interface_tutorial.cpp

```diff
@@ -452,6 +452,56 @@ int main(int argc, char** argv)
   // .. image:: ./move_group_interface_tutorial_attached_object.gif
   //    :alt: animation showing the arm moving differently once the object is attached
   //
+
+  // Moving to a goal pose with orientation tolerances
+  // ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  //
+  // Show text in RViz of status
+  visual_tools.deleteAllMarkers();
+  visual_tools.publishText(text_pose, "Goal with orientation tolerance", rvt::WHITE, rvt::XLARGE);
+  visual_tools.trigger();
+  // Let's plan to a goal colliding on the top of the collision object.
+  move_group_interface.setStartState(*move_group_interface.getCurrentState());
+  geometry_msgs::Pose pose_above_collision_object;
+  pose_above_collision_object.orientation.x = 1.0;
+  pose_above_collision_object.position.x = 0.45;
+  pose_above_collision_object.position.y = 0.0;
+  pose_above_collision_object.position.z = 0.59;
+  move_group_interface.setPoseTarget(pose_above_collision_object);
+
+  std::vector<double> goal_orientation_tolerance_xyz = { 1e-3, M_PI / 2, M_PI / 4 };
+  move_group_interface.setGoalOrientationToleranceXYZ(goal_orientation_tolerance_xyz);
```

Comment:
I wasn't able to get this to build due to the "setGoalOrientationToleranceXYZ" method not being in the move_group_interface. Is this new functionality that also needs to be added?